[2024-09-20T20:44:07.656+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-20T20:44:07.667+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: primeira_dag_versao_2.terceira_task scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-09-20T20:44:07.674+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: primeira_dag_versao_2.terceira_task scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-09-20T20:44:07.674+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-20T20:44:07.682+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): terceira_task> on 2024-09-16 00:00:00+00:00
[2024-09-20T20:44:07.688+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=243) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-20T20:44:07.689+0000] {standard_task_runner.py:72} INFO - Started process 254 to run task
[2024-09-20T20:44:07.689+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'primeira_dag_versao_2', 'terceira_task', 'scheduled__2024-09-16T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/primeira_dag.py', '--cfg-path', '/tmp/tmp2fi67m2p']
[2024-09-20T20:44:07.690+0000] {standard_task_runner.py:105} INFO - Job 46: Subtask terceira_task
[2024-09-20T20:44:07.717+0000] {task_command.py:467} INFO - Running <TaskInstance: primeira_dag_versao_2.terceira_task scheduled__2024-09-16T00:00:00+00:00 [running]> on host 22a02bcf3677
[2024-09-20T20:44:07.768+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Everton Soares' AIRFLOW_CTX_DAG_ID='primeira_dag_versao_2' AIRFLOW_CTX_TASK_ID='terceira_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-16T00:00:00+00:00'
[2024-09-20T20:44:07.768+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-20T20:44:07.779+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-20T20:44:07.779+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Olá, sou a terceira tarefa'"]
[2024-09-20T20:44:07.788+0000] {subprocess.py:86} INFO - Output:
[2024-09-20T20:44:07.789+0000] {subprocess.py:93} INFO - Olá, sou a terceira tarefa
[2024-09-20T20:44:07.790+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-20T20:44:07.806+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-20T20:44:07.806+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=primeira_dag_versao_2, task_id=terceira_task, run_id=scheduled__2024-09-16T00:00:00+00:00, execution_date=20240916T000000, start_date=20240920T204407, end_date=20240920T204407
[2024-09-20T20:44:07.822+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-20T20:44:07.832+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-20T20:44:07.833+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
